logo/signet/signet_default.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="103" height="114" viewBox="0 0 103 114" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Just wondering, maybe it would be better to export it as a square? Or add the square viewBox as another variant.

Reply:
I checked that on my software because sometimes vector rendering is broken but the shape is actually ok, but this one seem broken